skinlist.lua

```diff
@@ -52,8 +56,9 @@ for _, fn in pairs(skins_dir_list) do
 	end
 end
 
--- get skinlist. If assignment given ("mod:wardrobe" or "player:bell07") select skins matches the assignment. select_unassigned selects the skins without any assignment too
+-- (obsolete) get skinlist. If assignment given ("mod:wardrobe" or "player:bell07") select skins matches the assignment. select_unassigned selects the skins without any assignment too
 function skins.get_skinlist(assignment, select_unassigned)
+	minetest.log("deprecated", "skins.get_skinlist() is deprecated. Use skins.get_skinlist_for_player() instead")
```

Comment:
Non-existent log level
>  * `level` is one of `"none"`, `"error"`, `"warning"`, `"action"`,
> `"info"`, or `"verbose"`. Default is `"none"`.

Reply:
I found thte loglevel on http://dev.minetest.net/minetest.log
Wich loglevel is usual for deprecation messages?

Reply:
Never mind. This time it's `lua_api.txt` that is inaccurate. `deprecated` is a special level for the function `minetest.log`, so it's fine to use it here.